myscripts.js

```diff
@@ -29,3 +30,102 @@ menuLink3.addEventListener('click', () => {
   menuButton.classList.remove('isActive');
   menu.classList.remove('isActive');
 });
+
+const projectsDetails = [
+  {
+    id: 1,
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['CSS', 'HTML', 'RUBY', 'BOOTSTRAP'],
+    image: './images/number1.jpg',
+    link: 'https://github.com/ReAliens/Portofilo',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['CSS', 'HTML', 'RUBY', 'BOOTSTRAP'],
+    image: './images/number2.jpg',
+    link: 'https://github.com/ReAliens/Portofilo',
+  },
+  {
+    id: 3,
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['CSS', 'HTML', 'RUBY', 'BOOTSTRAP'],
+    image: './images/number3.jpg',
+    link: 'https://github.com/ReAliens/Portofilo',
+  },
+  {
+    id: 4,
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['CSS', 'HTML', 'RUBY', 'BOOTSTRAP'],
+    image: './images/number4.jpg',
+    link: 'https://github.com/ReAliens/Portofilo',
+  },
+];
+
+projectsDetails.map((project) => {
+  const projectSummary = `
+  <section class='work ws-${project.id}' id='third-section'>
+  <img src='${project.image}' alt='cover' />
+  <div class='work-story'>
+    <h2>${project.name}</h2>
+    <p>
+   ${project.description}
+    </p>
+    <ul class='w-section-tech'>
+    ${project.technologies
+    .map((item, index) => `<li class=${index === 3 ? 'last-tech' : ''}>${item}</li>`)
+    .join('')}
+    </ul>
+    <button data-modal=${
+  project.id
+} type='button' class='viewButton'>See Project</button>
+</section>`;
+  return workPart.insertAdjacentHTML('beforeend', projectSummary);
+});
```

Comment:
- [ ] You might want to consider running this part when the page loads. Some resources to do it 📚 
- [DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event)
- [onload](https://developer.mozilla.org/es/docs/Web/API/GlobalEventHandlers/onload)

I saw that you try to use it here
https://github.com/ReAliens/Portofilo/blob/6d69f1eb4ec618064c726f2807c67bbe32b0d794/index.html#L33
However, there isn't any `addElements()` function in your JavaScript file.